pkg/sensors/program/loader.go

```diff
@@ -139,6 +149,11 @@ func LoadKprobeProgram(bpfDir, mapDir string, load *Program, verbose int) error
 	return loadProgram(bpfDir, []string{mapDir}, load, KprobeAttach(load), ci, verbose)
 }
 
+func LoadTailCallProgram(bpfDir, mapDir string, load *Program, verbose int) error {
+	ci := &customInstall{fmt.Sprintf("%s-tail-call", load.PinPath), "kprobe"}
```

Comment:
maybe %s-tail-calls with s?

Reply:
sure, will update that now

Reply:
maybe I can remove this line entirely and use nil in a similar way to `LoadRawTracepointProgram`. Checking that now.

Reply:
removed that entirely